performance/index.ts

```diff
@@ -2,118 +2,106 @@
 //Options:
 // run transfer
 
-//npx ts-node ./performance/index.ts transfer threadNumber=3 testCaseName=transfer duration=1000 totalTransactions=25000 nodes=ws://ws1,ws://ws2
+//npx ts-node ./performance/index.ts threadNumber=3 testCaseName=transfer duration=1000 totalTransactions=25000 nodes=ws://ws1,ws://ws2
 
 import {TestParams} from "./testParams";
 import {Tests, TestFactory} from "./testFactory";
 
 async function main() {
   const args = process.argv.slice(2);
-  const command = args[0];
-  const commandArguments = args.slice(1);
-  switch (command) {
-    case "transfer":
```

Comment:
This is fine _if_ we don't want the ability to easily extend the cli in the future. This pattern is for a multi-command application where you have `cli <command> [args]` and the change makes it a single-command application with `cli [args]`. Might not be needed so up to you!